docs/_writing_dependencies_conda.rst

```diff
@@ -0,0 +1,316 @@
+it to add tools to the existing channels.
+
+The existing channels include:
+
+* Bioconda (`github <https://github.com/bioconda/bioconda-recipes>`__ | `conda <https://anaconda.org/bioconda>`__) - best practice channel for various bioinformatics packages.
+* Conda-Forge (`github <https://github.com/conda-forge/staged-recipes>`__ | `conda <https://anaconda.org/conda-forge>`__) - best practice channel for general purpose and widely useful computing packages and libraries.
+* iuc (`github <https://github.com/galaxyproject/conda-iuc>`__ | `conda <https://anaconda.org/iuc>`__) - best practice channel for other more Galaxy specific packages.
+
+----------------------------------------------------------------
+Exercise - Leveraging Bioconda
+----------------------------------------------------------------
+
+Use the ``project_init`` command to download this exercise.
+
+::
+
+    $ planemo project_init --template conda_exercise conda_exercise
+    $ cd conda_exercise
+    $ ls 
+    pear.xml              test-data
+
+This will download a tool for `PEAR - Paired-End reAd mergeR
+<http://sco.h-its.org/exelixis/web/software/pear/>`__. This tool however has
+``requirement`` tags and so will not work properly.
+
+1. Run ``planemo test pear.xml`` to verify the tool does not function
+   without dependencies defined.
+1. Use ``--conda_requirements`` flag with ``planemo lint`` to verify it does
+   indeed lack requirements.
+1. Use ``planemo conda_search`` or the Anaconda_ website to search for the
+   correct package and version in a best practice channel.
+1. Update ``pear.xml`` with the correct ``requirement`` tags.
+1. Re-run the ``lint`` command from above to verify the tool now has the
+   correct dependency definition.
+1. Re-run the ``test`` command from above to verify the tool test now
+   works properly.
+
+----------------------------------------------------------------
+Building New Conda Packages
+----------------------------------------------------------------
+
+Frequently packages your tool will require are not found in Bioconda_
+or conda-forge yet. In these cases, it is likely best to contribute
+your package to one of these projects. Unless the tool is exceedingly
+general Bioconda_ is usually the correct starting point.
+
+.. note:: Many things that are not strictly or even remotely "bio" have
+    been accepted into Bioconda_ - including tools for image analysis,
+    natural language processing, and cheminformatics.
+
+At this time, the most relevant source for information on building Conda packages for Galaxy
+is probably the Bioconda_ documentation - in particular check out the `contributing documentation
+<https://bioconda.github.io/contributing.html>`__.
+
+----------------------------------------------------------------
+Exercise - Package a Tool
+----------------------------------------------------------------
+
+1. Package a tool for Bioconda.
```

Comment:
I know I would get push back if I put it here - so this is probably not going to end up in the planemo docs. 😢 If it does - it might be a smaller section on galaxy-specific or planemo-specific tricks that already assumes the reader has gone through bioconda docs.